app/models/order.rb

```diff
@@ -1,5 +1,20 @@
 class Order < ApplicationRecord
   has_many :order_items
+  validates :name, :email, :mailing_address, :cc_name, :cc_cvv, :cc_exp, :cc_number, :zipcode, :status, presence: true
+  validates :email, format: { with: /\w+@\w+\.\w+/, message: "Please enter valid email address" }
+  validates :cc_cvv, :cc_number, :zipcode, numericality: { only_integer: true }
+  validates :cc_number, length: {is: 15}
+  validates :cc_cvv, length: {is: 3}
+  validates :zipcode, length: {is: 5}
 
```

Comment:
cc_number length validation should be 16? Once we merge, let's check if the tests and orders#show need to be revised.